test/routes/home_route_test.dart

```diff
@@ -50,6 +50,8 @@ void main() {
     await tester.runAppTest(() async {
       expect(Permissions.instance.granted, true);
       expect(find.byType(Home), findsOneWidget);
+      expect(tester.getRect(find.byType(TrackShowcase)).top,
+          tester.getRect(find.byType(App)).height, reason: 'Player route must be offscreen');
```

Comment:
This test fails without the fix to the `AppBar` height. It should prevent regressions in the future.

Reply:
This should rather verify that the player route is actually offscreen

Reply:
I tried that, but
```dart
expect(find.byType(TrackShowcase), findsNothing, reason: 'Player route must be offscreen');
```
fails because the finder still finds the TrackShowcase (even though `skipOffstage` is `true`).

Do you have a better way to test that it's offscreen?

Reply:
Seems like getting its rect and expecting it to fail should do the thing

Reply:
Actually no, `hitTestable` can't be used to filter it. Do you have another way to get the rect that fails, when the widget is offscreen?

Reply:
It's good to me like it is then, but reformat it to 3 separate lines by arguments, it would look cleaner